guide/popular-topics/threads.md

```diff
@@ -0,0 +1,153 @@
+# Threads
+
+Threads can be thought of as temporary sub-channels inside an existing channel, to help better organize conversation in a busy channel.
+
+## Thread related gateway events
+
+::: tip
+You can use the `isThread()` typeguard to make sure the channel you're working with is a ThreadChannel!
+:::
+
+Threads introduce a number of new gateway events, which are listed below:
+
+- `Client#threadCreate`: Emitted whenever a thread is created or when the client user is added to a thread.
+- `Client#threadDelete`: Emitted whenever a thread is deleted.
+- `Client#threadUpdate`: Emitted whenever a thread is updated - e.g. name change, archive state change, locked state change.
+- `Client#threadListSync`: Emitted whenever the client user gains access to a text or news channel that contains threads.
+- `Client#threadMembersUpdate`: Emitted whenever members are added or removed from a thread. Requires `GUILD_MEMBERS` privileged intent.
+- `Client#threadMemberUpdate`: Emitted whenever the client user's thread member is updated.
+
+## Creating and deleting threads
+
+Threads are created and deleted using the `ThreadManager` of a text or news channel.
+To create a thread you call the `ThreadManager#create()` method:
+
+```js {2-8}
+client.on('messageCreate', async message => {
+	const thread = await message.channel.threads
+		.create({
+			name: 'food-talk',
+			autoArchiveDuration: 60,
+			reason: 'Needed a separate thread for food',
+		});
+	console.log(`Created thread: ${thread.name}`);
+});
+```
+
+To delete a thread you call the `ThreadChannel#delete()` method:
+
+```js {2,3}
+client.on('messageCreate', async message => {
+	const thread = message.channel.threads.cache.find(x => x.name === 'food-talk');
+	await thread.delete();
+});
+```
+
+## Joining and leaving threads
+
+To join your Client to a ThreadChannel you make use of the `ThreadChannel#join()` method:
```

Comment:
Either here or in the private vs public section something about checking whether the thread is joinable maybe? (really only matters for private)